### PR TITLE
Update Boston Hackathon

### PR DESCRIPTION
**Pull request by christopher-hakkaart**

### CODE REVIEW

File: src/content/events/2023/hackathon-november-2023.mdx [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+import Profile from '@components/GitHubProfilePictureExtended.svelte';
+import HackathonGroup from '@components/event/HackathonGroup.astro';
+import { YouTube } from '@astro-community/astro-embed-youtube';
+
+# Welcome
+
+Join us in person for the November 2023 nf-core hackathon!
+
+This hackathon will be held in advance of the _**Nextflow Summit 2023**_ in Boston, America 🇺🇸. You can find out more information about the summit at <https://summit.nextflow.io>.
```
Comment (mashehu) [inline src/content/events/2023/hackathon-november-2023.mdx:22]:
```suggestion
This hackathon will be held in advance of the _**Nextflow Summit 2023**_ in Boston, America 🇺🇸. You can find out more information about the summit at [https://summit.nextflow.io](https://summit.nextflow.io).
```

File: src/content/events/2023/hackathon-november-2023.mdx [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+This hackathon is not meant to be a training event and is aimed at people that already have an understanding of Nextflow and/or nf-core.
+If you're interested in learning Nextflow and nf-core, there are recordings of online workshops from September 2023 that cover foundational and advance Nextflow usage.
+See the [Community Foundational Nextflow Training](https://nf-co.re/events/2023/training-basic-2023) and [Community Advanced Nextflow Training](https://nf-co.re/events/2023/training-sept-2023) pages for more information and links to the training material.
+
+# Registration
+
+You can register to attend the hackathon in-person using the links on the Nextflow Summit website: <https://summit.nextflow.io/register>
```
Comment (mashehu) [inline src/content/events/2023/hackathon-november-2023.mdx:30]:
```suggestion
You can register to attend the hackathon in-person using the links on the [Nextflow Summit website](https://summit.nextflow.io/register)
```

**Review by mashehu — APPROVED** [review]

File: src/content/events/2023/hackathon-november-2023.mdx (hunk 2 above)
Comment (christopher-hakkaart) [reply src/content/events/2023/hackathon-november-2023.mdx]: Ahh thanks!

Comment (ewels) [reply src/content/events/2023/hackathon-november-2023.mdx]: @christopher-hakkaart note that this is an MDX specific thing - the syntax you used is valid Markdown and would be find in `.md` files. But in `.mdx` it gets incorrectly parsed as a custom element tag and breaks everything.

**Pull request merged**